src/registrar/templates/home.html

```diff
@@ -111,7 +111,7 @@ <h2>Domain requests</h2>
             {{ application.requested_domain.name|default:"New domain request" }}
           </th>
           <td data-sort-value="{{ application.created_at|date:"U" }}" data-label="Date created">{{ application.created_at|date }}</td>
-          <td data-label="Status">{{ application.status|title }}</td>
+          <td data-label="Status">{{ application.status|capfirst }}</td>
```

Comment:
Maybe try application.get_status_display() which might be marginally better than getting the value and putting a filter on it.